internal/scorecard/scorecard_output.go

```diff
@@ -62,7 +62,7 @@ func printPluginOutputs(version string, pluginOutputs []scapiv1alpha1.ScorecardO
 	return nil
 }
 
-func combinePluginOutput(pluginOutputs []scapiv1alpha1.ScorecardOutput) (scapiv1alpha1.ScorecardOutput, error) {
+func combinePluginOutput(cfg Config, pluginOutputs []scapiv1alpha1.ScorecardOutput) (scapiv1alpha1.ScorecardOutput, error) {
```

Comment:
This could be a method on `Conifg`.

Reply:
good suggestion, the change will be in the next commit.